public/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1, maximum-scale=1">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <link rel="stylesheet" href="css/main.css">
+    <title>Payments UI</title>
+  </head>
+  <body data-static-url='.'>
+    <main>
+      <div id='view'></div>
+    </main>
+    <script src="js/bundle.js"></script>
```

Comment:
how does this work for development? Will we see tracebacks to the original source files?